CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org).
 
+## Supported Release [2.3.0]
```

Comment:
This 2.3.0 link isn't an actual link like the others.

Reply:
@ekohl we will likely fix it in master for the next release. apologies. 

and thanks for checking !!! 👍